react-with-graphql-tutorial.md

```diff
@@ -260,9 +263,9 @@ Whereas you would have to provide the second variable to request a specific repo
 }
 {{< /highlight >}}
 
-The organization that teaches about React has translated versions of its content. One repository teaches students about React in simplified Chinese. As you can see, fields in GraphQL can be nested objects again. Basically, you have queried two associated objects from the graph. After all, you are making requests on a graph which can have a deeply nested structure. As you have explored the "Docs" sidebar in GraphiQL before, you might have seen that you can jump from object to object in the graph.
+The organization that teaches about React has translated versions of its content, and one of its repositories teaches students about React in simplified Chinese. Fields in GraphQL can be nested objects again, and you have queried two associated objects from the graph. The requests are made on a graph that can have a deeply nested structure. While exploring the "Docs" sidebar in GraphiQL before, you might have seen that you can jump from object to object in the graph.
 
-There is one last missing piece for structuring your queries in GraphQL: **directives**. A directive can be used to query conditional data from your GraphQL API. This way, you can apply conditional structures to your shape of query. They can be applied to fields and objects. You can either use the **include directive** or **skip directive**. Whereas the include directive only includes the field when the `Boolean` type is set to true, the skip directive excludes the field instead. The following query showcases the include directive, but you can substitute it with the skip directive on your own.
+A **directive** can be used to query conditional data from your GraphQL API, so you can apply conditional structures to your shape of query. Directives can also be applied to fields and objects.  Below, we use two types: an **include directive**, which includes the field when the `Boolean` type is set to true; and the skip directive, which excludes it instead. The following query showcases the include directive, but you can substitute it with the skip directive to achieve the opposite effect:
```

Comment:
Explanation is not totally right, because not every directive is used for **conditional data**, but by restructuring the sentences a bit, we should be alright:

A **directive** can be used to query data in a more powerful way from your GraphQL API. They can be applied to fields and objects.  Below, we use two types of directives: an **include directive**, which includes the field when the `Boolean` type is set to true; and the **skip directive**, which excludes it instead. With the help of these directives, you can apply conditional structures to your shape of query. The following query showcases the include directive, but you can substitute it with the skip directive to achieve the opposite effect: